<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Captcha_generated">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>92</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>List of files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnObj">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>92</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnScript">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>92</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Script</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPcapOp">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>360</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pcap output</string>
    </property>
   </widget>
   <widget class="QLabel" name="ToolName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Automated tool </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string> Enter .pcap file :</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPrerequi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Prerequisites:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRequi1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>User must have network captured(.pcap) file</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRequi2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>341</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>User need to install pcap parser first</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Result : </string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelResultMsg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Result to shown here ...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LineEditFname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
